test/controllers/confirmation.controller.spec.ts

```diff
@@ -36,6 +36,7 @@ describe("Confirmation controller tests", () => {
     expect(resp.status).toEqual(200);
     expect(resp.text).toContain(CONFIRMATION_PAGE_TITLE);
     expect(resp.text).toContain(TRANSACTION_ID);
+    expect(resp.text).toContain("within 2 working days");
```

Comment:
Not a blocker, you can export the `"within 2 working days"` to the mock text file

Reply:
Thanks @mwestacott for the change